secure-storage-sample/index.js

```diff
@@ -0,0 +1,20 @@
+const secureStorage = require("uxp").storage.secureStorage;
+const app = window.require("photoshop").app;
+
+
+async function saveSecureKey() {
+    const value = document.getElementById("secure").value;
+    await secureStorage.setItem("secure", value);
+    document.getElementById("secure").value = ""
+}
+
+async function fetchSecureKey() {
+  const value = await secureStorage.getItem("secure");
+  let string = "";

```

Comment:
Let's add a comment that illustrates that secureStorage is returning a uInt8Array, and that 13-14 is building the string out of it.